tests/security_features_test.go

```diff
@@ -141,6 +142,30 @@ var _ = Describe("SecurityFeatures", func() {
 				}
 				Expect(*container.SecurityContext.Privileged).To(BeFalse())
 			})
+
+			It("[test_id:5297]Make sure qemu processes are MCS constrained", func() {
+
+				By("Starting a VirtualMachineInstance")
+				vmi, err = virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Create(vmi)
+				Expect(err).ToNot(HaveOccurred())
+				tests.WaitForSuccessfulVMIStart(vmi)
+
+				pod := tests.GetRunningPodByVirtualMachineInstance(vmi, tests.NamespaceTestDefault)
+
+				qemuProcess, err := tests.ExecuteCommandOnPod(
+					virtClient,
+					pod,
+					"compute",
+					[]string{"/usr/bin/bash", "-c", "ps -efZ | grep [q]emu-kvm | awk '{print $1}'"},
+				)
+				Expect(err).ToNot(HaveOccurred())
+
+				By("Checking that qemu-kvm process is of the SELinux type container_t")
```

Comment:
Qemu processes should run as container_t only if `selinuxLauncherType` is not set in the kubevirt-config configmap.
Otherwise, the domain on qemu processes should be equal to the value of `selinuxLauncherType`.

Reply:
yeah, agreed. hence this test-case is under `Context("With selinuxLauncherType undefined"`,  in kubevirt-config CM.

Reply:
Ah, missed that, thanks!